### PR TITLE
Gradle Enterprise is renamed to Develocity

### PR DESCRIPTION
**Pull request by runningcode**
This renames Gradle Enterprise to Develocity in the README and no other files.

### CODE REVIEW

File: Maven.md [hunk 1]
```diff
@@ -129,20 +129,20 @@ The scripts that fetch build scan data expect some of it to be present as custom
 ## Redirecting build scan publishing
 
 The scripts that run one or more builds locally can be configured to publish build scans to a different
-Gradle Enterprise server than the one that the builds point to by passing the `-s` or `--gradle-enterprise-server`
+Develocity server than the one that the builds point to by passing the `-s` or `--gradle-enterprise-server`
 command line argument. In the example below, the script will configure the local builds to publish their build scans
 to ge.example.io regardless of what server is configured in the build.
 
 ```bash
 ./01-validate-local-build-caching-same-location.sh -i -s https://ge.example.io
```
Comment (jthurne) [inline Maven.md:137]:
```suggestion
./01-validate-local-build-caching-same-location.sh -i -s https://develocity.example.io
```

Probably would be worth doing a find-replace on `ge.example.io`.

**Review by jthurne — APPROVED** [review]

File: Maven.md (hunk 1 above)
Comment (runningcode) [reply Maven.md]: Done!

File: Gradle.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ If you plan to use the build validation scripts on Windows, then you will need t
 
 ## Compatibility
 
-The build validation scripts are compatible with a large range of Gradle versions, as laid out in the table below. Getting the best user experience when running an experiment and when being presented with the results of an experiment requires access to the Gradle Enterprise server that holds the captured build data. Fetching that build data requires a compatible version of Gradle Enterprise, as laid out in the table below.
+The build validation scripts are compatible with a large range of Gradle versions, as laid out in the table below. Getting the best user experience when running an experiment and when being presented with the results of an experiment requires access to the Develocity server that holds the captured build data. Fetching that build data requires a compatible version of Develocity, as laid out in the table below.
 
-| Build Validation Scripts version | Compatible Gradle versions | Compatible Gradle Enterprise versions |
+| Build Validation Scripts version | Compatible Gradle versions | Compatible Develocity versions |
```
Comment (erichaagdev) [inline Gradle.md:19]: Can you reformat the table?

File: Maven.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ If you plan to use the build validation scripts on Windows, then you will need t
 
 ## Compatibility
 
-The build validation scripts are compatible with a large range of Maven versions, as laid out in the table below. Getting the best user experience when running an experiment and when being presented with the results of an experiment requires access to the Gradle Enterprise server that holds the captured build data. Fetching that build data requires a compatible version of Gradle Enterprise, as laid out in the table below.
+The build validation scripts are compatible with a large range of Maven versions, as laid out in the table below. Getting the best user experience when running an experiment and when being presented with the results of an experiment requires access to the Develocity server that holds the captured build data. Fetching that build data requires a compatible version of Develocity, as laid out in the table below.
 
-| Build Validation Scripts version | Compatible Maven versions | Compatible Gradle Enterprise versions |
+| Build Validation Scripts version | Compatible Maven versions | Compatible Develocity versions |
```
Comment (erichaagdev) [inline Maven.md:19]: Can you reformat the table?

**Review by erichaagdev — CHANGES_REQUESTED** [review]

File: Gradle.md (hunk 2 above)
Comment (runningcode) [reply Gradle.md:19]: thanks

**Pull request merged**